src/com/company/EnemyTank.java

```diff
@@ -7,18 +7,78 @@
  * Created by akupp_000 on 5/8/2017.
  */
 public class EnemyTank extends Tank {
+	
+	private Handler handler;
+	
     public EnemyTank(float x, float y, ObjectId id, Handler handler){
         super(x,y,id, handler);
+        this.handler = handler;
     }
     @Override
     public void tick(LinkedList<GameObject> object) {
-
+    	Collision(object);
     }
-
+    
+    public void Collision(LinkedList<GameObject> object) {
+    	for(int i=0; i < handler.object.size();i++)
+    	{
+    		GameObject tempObject = handler.object.get(i);
+    		
+    		if(tempObject.getID() == ObjectId.Block)
+    		{
+    			if(getBoundsTop().intersects((tempObject.getBounds())))
+    			{
+    				y = tempObject.getY() + (height/2);
+    				velY= 0;
+    			}
+    			
+    			if(getBounds().intersects((tempObject.getBounds())))
+    			{
+    				y = tempObject.getY() - height;
+    				velY= 0;
+    			}	
+    			
+    			if(getBoundsRight().intersects((tempObject.getBounds())))
+    			{
+    				x = tempObject.getX() - width;
```

Comment:
The inner statements are the same (`x = tempObject.getX() - width;`), why do you separate them into 2 different `if` blocks?